pkg/ebpf/gadgets/http/tracer/bpf/http-sniffer.c

```diff
@@ -0,0 +1,623 @@
+#include "sniffer_strcuts.h"
```

Comment:
First builtin includes.

Reply:
DONE

Reply:
The builtins needs sniffer_strcuts in order to be included